core/pages/Category.js

```diff
@@ -252,6 +253,15 @@ export default {
           this.$store.dispatch('category/products', this.getCurrentCategoryProductQuery)
           EventBus.$emitFilter('category-after-load', { store: this.$store, route: this.$route })
         }
+      }).catch(err => {
+        if (err.message.indexOf('query returned empty result') > 0) {
```

Comment:
please move notification to theme so it can be configured (either use an overridable method for this or a callback)

Reply:
This is not a change from this issue :)